lib/util/search/start_destination_timeline.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:timelines/timelines.dart';
+
+import '../custom_timeline_theme.dart';
+import 'address_search_field.dart';
+import 'address_suggestion.dart';
+
+class StartDestinationTimeline extends StatelessWidget {
+  final TextEditingController startController;
+  final TextEditingController destinationController;
+  final void Function(AddressSuggestion) onStartSelected;
+  final void Function(AddressSuggestion) onDestinationSelected;
+
+  const StartDestinationTimeline({
+    super.key,
+    required this.startController,
+    required this.destinationController,
+    required this.onStartSelected,
+    required this.onDestinationSelected,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return FixedTimeline(
+      theme: CustomTimelineTheme.of(context),
+      children: <Widget>[
+        TimelineTile(
+          contents: Padding(
+            padding: const EdgeInsets.all(4.0),
```

Comment:
can we add a little bit more Padding to the left. "Das braucht Platz zum Atmen" zitat by Merri